designer/src/components/section-editor.tsx

```diff
@@ -52,36 +58,68 @@ type Props = {
     viewID: string,
     moveDirection: 'left' | 'right'
   ) => void;
+  moveSectionCallback: (
+    sourceViewSetID: string,
+    targetViewSetID: string,
+    viewID: string
+  ) => boolean;
 };
 
 export const SectionEditor = ({
   viewSetId,
   viewId,
   viewSet,
   deleteCallback,
+  moveSectionCallback,
   addCallback,
   moveCallback,
 }: Props) => {
   const fView = useAppSelector(
     state => state.notebook['ui-specification'].fviews[viewId]
   );
+  const viewSets = useAppSelector(
+    state => state.notebook['ui-specification'].viewsets
+  );
   const dispatch = useAppDispatch();
 
   console.log('SectionEditor', viewId, viewSet);
 
-  const [open, setOpen] = useState(false);
+  const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
+  const [openMoveDialog, setOpenMoveDialog] = useState(false);
+  const [targetViewSetId, setTargetViewSetId] = useState('');
   const [editMode, setEditMode] = useState(false);
   const [addMode, setAddMode] = useState(false);
   const [newSectionName, setNewSectionName] = useState('New Section');
   const [addAlertMessage, setAddAlertMessage] = useState('');
 
-  const handleClose = () => {
-    setOpen(false);
+  const handleCloseDeleteDialog = () => {
+    setOpenDeleteDialog(false);
+  };
+
+  const handleCloseMoveDialog = () => {
+    setOpenMoveDialog(false);
   };
 
   const deleteSection = () => {
     deleteCallback(viewSetId, viewId);
-    handleClose();
+    handleCloseDeleteDialog();
+  };
+
+  const moveSectionToForm = () => {
+    // run the function to move the section to a different form AND save the returned success status to a variable
+    const moveSuccess: boolean = moveSectionCallback(viewSetId, targetViewSetId, viewId);
+
+    // depending on moveSuccess, set relevant state variables
+    if (moveSuccess) {
```

Comment:
We should show some kind of message when it works. I also thought that it might be better if we navigate to view the section in it's new form, not sure how feasible that would be  since at the moment we don't encode the current section in the URL.  At least navigating to the new form would be useful I think.